okonomiyaki/file_formats/_egg_info.py

```diff
@@ -820,6 +881,22 @@ def egg_name(self):
         return self._spec_depend.egg_name
 
     @property
+    def is_strictly_supported(self):
+        """ Returns True if the given metadata_version is fully supported.
+
+        A metadata_version is fully supported iff:
+            - metadata_version.major ==
+              EggMetadata.HIGHEST_SUPPORTED_METADATA_VERSION.major
+            - and metadata_version.minor <=
+              EggMetadata.HIGHEST_SUPPORTED_METADATA_VERSION.minor
+        """
+        max_supported = EggMetadata.HIGHEST_SUPPORTED_METADATA_VERSION
+        return (
+            _are_compatible(self.metadata_version, max_supported) and
```

Comment:
This seems to imply that when we support metadata version 2.0, we will not support 1.x. Is that the case, or will 2.x support be through an entirely different interface?